### ISSUE TITLE
GTX 1070 rig crashes after creating binfile.

### ISSUE THREAD

**Issue by lyolyalya** [zawawawa/gatelessgate#6]
Hey.have one rig wit 3x1070 GTX.after i start zcash(any platform) my rig crashes and rebooting?can you help to resolve this problem?any other mining software rock and solid for days.

**Comment by zawawawa** [comment]
OS and driver version?

**Comment by lyolyalya** [comment]
win 10х64, latest nvidia drivers. but this problems were earlier too...

**Comment by zawawawa** [comment]
Any BIOS mods or overclocking/underclocking?

**Comment by lyolyalya** [comment]
no bios modding.slight OC but again - all other miners are working fine...

**Comment by lyolyalya** [comment]
tried without OC - zcash - starts but only 40% of using GPU.pascal crashes.

**Comment by zawawawa** [comment]
How about CPU usage?

**Comment by lyolyalya** [comment]
yes.cpu is up to 96% with three cards on zcash.sorry.o forgot it...it was even on ethminer with opencl...so?no possibility to run your miner on a crappy cpu?=(

**Comment by zawawawa** [comment]
This issue is a long-standing one, and I just need to improve the workaround I already implemented. Let's see what I can do.

**Comment by lyolyalya** [comment]
okay.will wait.let us know if something changed.cheers
